test-requirements.txt

```diff
@@ -9,6 +9,7 @@ flake8-noqa==1.3.1; python_version >= "3.8"      # must match version in .pre-co
 isort[colors]==5.12.0; python_version >= "3.8"   # must match version in .pre-commit-config.yaml
 lxml>=4.9.1; (python_version<'3.11' or sys_platform!='win32') and python_version<'3.12'
 pre-commit
+pre-commit-hooks==4.4.0
```

Comment:
I guess we can add this for consistency, but I don't think people will use pre-commit-hooks as a standalone package. I didn't even know it could be run standalone until now.

Reply:
Yeah, it does seem unlikely that people are going to be manually running `trailing whitespace mypy/semanal.py` locally. And it's true that pre-commit requirements don't need to be pre-installed for pre-commit to work locally. So, I'm happy to remove this -- but curious to hear what others think, as well.